### ISSUE TITLE
No assets are exported to target build folder

### ISSUE THREAD

**Issue by RuiCuco** [Kode/Kha#1132]
When building project, no assets in 'assets' folder are exported to target build folder. This is the khafile.js:

```
let project = new Project('MyProject');

project.addAssets('assets/**');
project.addSources('sources');

resolve(project);
```

**Comment by RobDangerous** [comment]
Please provide a complete project, I can't reproduce the issue like that, the khafile itself looks fine.

**Comment by RuiCuco** [comment]
Sorry. In order to best reproduce the error, you'll probably need to download Kha from github (I did it on Friday with the command 'git clone --recursive https://github.com/Kode/Kha' mentioned in the forum), point VS Code to it, and not use your usual copy of Kha.

[TestKha.zip](https://github.com/Kode/Kha/files/3673140/TestKha.zip)

**Comment by RblSb** [comment]
Try latest Kha, should be already fixed.

**Comment by RobDangerous** [comment]
Yes, this was probably fixed in 30f0d6f5. Please always update Kha to the very latest version before you file an issue.

**Comment by RobDangerous** [comment]
Well, guess this is fixed.
